File: C Files/1-2/Exercises/average.c
#include <stdio.h>

int main() {
  int a, b, c;
  float avg;

  printf("Enter the first number: ");
  scanf("%d", &a);
  printf("Enter the second number: ");
  scanf("%d", &b);
  printf("Enter the third number: ");
  scanf("%d", &c);

  avg = (a + b + c) / 3.0;
  printf("The average of the three numbers is: %f\n", avg);

  return 0;
}

File: Java Files/MOOC/Part-I/Average.java
import java.util.Scanner;

public class Average {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    System.out.println("Give the first number:");
    String num1 = scanner.nextLine();

    System.out.println("Give the second number");
    String num2 = scanner.nextLine();
    double avg = (Double.parseDouble(num1) + Double.parseDouble(num2)) / 2;
    System.out.println("The average is " + avg);
    scanner.close();
  }
}
